layouts/fastly/helix.vcl

```diff
@@ -555,8 +581,8 @@ sub hlx_type_static_url {
     set var.static_version = {"const:static_version"};
   }
 
-  set req.http.X-Action-Root = "/api/v1/web/" + table.lookup(secrets, "OPENWHISK_NAMESPACE") + "/helix-services/static@" + var.static_version;
-  set req.http.X-Backend-URL = req.http.X-Action-Root
+  set req.http.X-Backend-URL = if(var.universal, "/", "/api/v1/web" + "/" + var.namespace)
```

Comment:
Add the `/api/v1/web/${namespace}` prefix for OpenWhisk – Universal Runtime does not need it.